new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -412,7 +469,43 @@ public void testVoyagerHostHttpRoutingAcrossDomains() {
     logger.info("Verifying http traffic");
     for (String domainUid : domainUids) {
       String ingressName = domainUid + "-ingress-host-routing";
-      verifyClusterLoadbalancing(domainUid, "http", getVoyagerLbNodePort(ingressName));
+      verifyClusterLoadbalancing(domainUid, defaultNamespace, "http", getVoyagerLbNodePort(ingressName),
+          replicaCount, true, "");
+    }
+  }
+
+  /**
+   * Verify Apache load balancer default sample through HTTP channel.
+   */
+  @Order(8)
+  @Test
+  @DisplayName("verify Apache load balancer default sample through HTTP channel")
+  public void testApacheDefaultSample() {
+
+    // verify Apache default sample
+    logger.info("Verifying Apache default sample");
+    int httpNodePort = getApacheNodePort(domain1Namespace, "http");
+    verifyClusterLoadbalancing(domain1Uid, domain1Namespace, "http", httpNodePort, 3, false, "/weblogic");
+  }
+
+  /**
+   * Verify Apache load balancer custom sample through HTTP and HTTPS channel.
```

Comment:
what is custom sample? explain the use case

Reply:
added more doc